spec/spec_helper.rb

```diff
@@ -35,5 +35,3 @@
 
 require 'webmock/rspec'
 WebMock.disable_net_connect!(allow: [/localhost/, /127\.0\.0\.1/])
-
-require 'zonebie/rspec'
```

Comment:
how come we removed this?

Reply:
it was causing deprecation-related errors, i think because it's not maintained. i removed it in favor of directly randomizing the timezone in `config/environments/test.rb`

```
  config.time_zone = ActiveSupport::TimeZone::MAPPING.keys.sample
  config.active_record.default_timezone = :local
```